sympy/vector/coordsys.py

```diff
@@ -95,55 +207,64 @@ def __new__(cls, name, location=None, rotation_matrix=None,
+            latex_vects = [(r'\mathbf{\hat{%s}_{%s}}' % (default_vector_names[coord_sys_type][0], name)),
+                           (r'\mathbf{\hat{%s}_{%s}}' % (default_vector_names[coord_sys_type][1], name)),
+                           (r'\mathbf{\hat{%s}_{%s}}' % (default_vector_names[coord_sys_type][2], name))]
+            pretty_vects = (name + '_' + default_vector_names[coord_sys_type][0],
+                name + '_' + default_vector_names[coord_sys_type][1], name + '_' + default_vector_names[coord_sys_type][2])
         else:
             _check_strings('vector_names', vector_names)
             vector_names = list(vector_names)
             latex_vects = [(r'\mathbf{\hat{%s}_{%s}}' % (x, name)) for
                            x in vector_names]
             pretty_vects = [(name + '_' + x) for x in vector_names]
 
-        obj._i = BaseVector(vector_names[0], 0, obj,
+        obj._e1cap = BaseVector(vector_names[0], 0, obj,
                             pretty_vects[0], latex_vects[0])
-        obj._j = BaseVector(vector_names[1], 1, obj,
+        obj._e2cap = BaseVector(vector_names[1], 1, obj,
                             pretty_vects[1], latex_vects[1])
-        obj._k = BaseVector(vector_names[2], 2, obj,
+        obj._e3cap = BaseVector(vector_names[2], 2, obj,
                             pretty_vects[2], latex_vects[2])
 
         # Initialize the base scalars
         if variable_names is None:
-            variable_names = (name + '.x', name + '.y', name + '.z')
-            latex_scalars = [(r"\mathbf{{x}_{%s}}" % name),
-                             (r"\mathbf{{y}_{%s}}" % name),
-                             (r"\mathbf{{z}_{%s}}" % name)]
-            pretty_scalars = (name + '_x', name + '_y', name + '_z')
+            variable_names = (name + '.' + default_scalar_names[coord_sys_type][0],
+                name + '.' + default_scalar_names[coord_sys_type][1], name + '.' + default_scalar_names[coord_sys_type][2])
+            latex_scalars = [(r'\mathbf{{%s}_{%s}}' % (default_scalar_names[coord_sys_type][0], name)),
+                            (r'\mathbf{{%s}_{%s}}' % (default_scalar_names[coord_sys_type][1], name)),
+                            (r'\mathbf{{%s}_{%s}}' % (default_scalar_names[coord_sys_type][2], name))]
+            pretty_scalars = (name + '_' + default_scalar_names[coord_sys_type][0],
+                name + '_' + default_scalar_names[coord_sys_type][1], name + '_' + default_scalar_names[coord_sys_type][2])
         else:
             _check_strings('variable_names', vector_names)
             variable_names = list(variable_names)
             latex_scalars = [(r"\mathbf{{%s}_{%s}}" % (x, name)) for
                              x in variable_names]
             pretty_scalars = [(name + '_' + x) for x in variable_names]
 
-        obj._x = BaseScalar(variable_names[0], 0, obj,
+        obj._e1 = BaseScalar(variable_names[0], 0, obj,
```

Comment:
`e1, e2, ...` etc. are generally used as base vector names. It would look better to use something else for coordinate scalars, e.g., `x1, x2, ...` (or `xi1, xi2, ...`).